RealmSwift/Tests/SortDescriptorTests.swift

```diff
@@ -37,4 +37,8 @@ class SortDescriptorTests: TestCase {
     func testDescription() {
         XCTAssertEqual(sortDescriptor.description, "SortDescriptor (property: property, direction: ascending)")
     }
+
+    func testStringLiteralConvertible() {
```

Comment:
You mean constructing a new `SortDescriptor` instead of using the already available `sortDescriptor` variable? All the other tests make use of the latter and it's there for a reason.